frontend/app/initializers/env-settings.js

```diff
@@ -0,0 +1,17 @@
+import config from "../config/environment";
+
+export function initialize(/* application */) {
+  /* eslint-disable no-undef  */
+  $.getJSON({
+    url: "/api/env_settings",
+    async: false,
```

Comment:
Sentry relies upon the synchronous call to the Backend to receive the dsn in the right moment. The problem is that this Mehthod is depracated. So I'm open for suggestions for a different solution.

Reply:
Don't really know a different way of handling this...